src/mcts/search.cc

```diff
@@ -682,7 +682,25 @@ void SearchWorker::ExtendNode(Node* node) {
       return;
     }
   }
-
+  
+  //WDL probing
+  if(board.our + board.their <= SyzygyTablebase::max_cardinality && board.GetNoCapturePly == 0
```

Comment:
count of total pieces hopefully :). i saw our + their used in board for checking sufficient material already

Reply:
not absolutely sure, but I think that this is the code that you are trying to call: https://github.com/LeelaChessZero/lc0/blob/2321011913d8a7914d8177e82ceaf34fbe2d6ee8/src/chess/board.h#L116
That return bitboards, so you will need to do "population count" (https://chessprogramming.wikispaces.com/Population+Count), I couldn't find a function for that in current lc0 code.

Reply:
popcount from stockfish: https://github.com/official-stockfish/Stockfish/blob/master/src/bitboard.h#L280 possibly going to need it implemented on https://github.com/LeelaChessZero/lc0/blob/master/src/chess/bitboard.h#L81

Reply:
I was trying to emulate this https://github.com/LeelaChessZero/lc0/blob/master/src/chess/board.cc#L707

Reply:
I see, but look at where those variables come from:
```
#ifdef _MSC_VER
  int our = _mm_popcnt_u64(our_pieces_.as_int());
  int their = _mm_popcnt_u64(their_pieces_.as_int());
#else
  int our = __builtin_popcountll(our_pieces_.as_int());
  int their = __builtin_popcountll(their_pieces_.as_int());
#endif
```
I think it would be good to refactor that code: move it to the Bitboard class, inside a `popcount` method.
That code would look like:
```
int our = our_pieces_.popcount();
int their = their_pieces_.popcount();
```
And the code here would be:
```
int our = board.ours().popcount();
int their = board.theirs().popcount();
```